docs/source/release_notes/release-notes.rst

```diff
@@ -146,12 +173,6 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2003>`__)
 
 
-    *    - |devbreak| |improved|
-         - Converted all compile time uses of Guava's ``com.google.common.base.Optional`` class to the Java8 equivalent ``java.util.Optional``.
-           Should not affect deployments as there is no change to `json` or `yml` representations of AtlasDB configurations.
-           This is a relatively straightforward compile time break for products consuming AtlasDB libraries.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2016>`__)
-
```

Comment:
I think the 0.44 is there, just not part of the diff that's shown (you'll need to expand it several times, unfortunately). I moved this one up because when we merged that PR the notes went into 44.

The release notes for 2014 are also present (just hidden because of the distance between the changes).

I'll redirect the change for 1891 to here.